### PR TITLE
Swap out the old contacts (Nunjucks) for the new contacts (React)

### PR DESCRIPTION
**Pull request by paulgain**
## Description of change

Swap out the old contacts implementation (Nunjucks) for the new (React)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/contacts/router.js [hunk 1]
```diff
@@ -30,20 +28,24 @@ const { getAudit } = require('./controllers/audit')
...
-  urls.contacts.index.route,
-  setDefaultQuery(DEFAULT_COLLECTION_QUERY),
-  getRequestBody(QUERY_FIELDS),
-  getCollection('contact', ENTITIES, transformContactToListItem),
-  renderContactList
-)
-
-// New react route (to replace the old companies list route above when complete)
-router.get(urls.contacts.react.index.route, renderContactsView)
+// TODO: Remove legacy contacts list code
```
Comment (paulgain) [inline src/apps/contacts/router.js:33]: This one's for you @elcct :wink:

File: test/sandbox/routes/v3/search/contact.js [hunk 2]
```diff
@@ -1,51 +1,12 @@
...
-  ) {
-    return res.json(contactFilter)
-  }
-
   if (req.body.original_query === 'no results') {
     return res.json({
       count: 0,
       results: [],
     })
   }
```
Comment (paulgain) [inline test/sandbox/routes/v3/search/contact.js:9]: We have to leave this in as a node test is dependent on it.

File: test/visual/wdio.conf.js [hunk 3]
```diff
@@ -34,14 +34,16 @@ exports.config = {
       'os': 'Windows',
       'os_version': '10',
       'browserName': 'Chrome',
-      'resolution': '1280x1024'
+      'resolution': '1280x1024',
+      'browserstack.timezone' : 'UTC'
```
Comment (ian-leggett) [inline test/visual/wdio.conf.js:38]: Nice 👍

**Review by ian-leggett — APPROVED** [review]
Looks good 👍

File: test/visual/wdio.conf.js (hunk 3 above)
Comment (paulgain) [reply test/visual/wdio.conf.js:38]: I got together with @PippoRaimondi on this one so can't take all the credit.

**Review by tomkingston — APPROVED** [review]

**Pull request merged**